exodus_lambda/functions/origin_request.py

```diff
@@ -19,6 +19,8 @@
 # this code against localstack.
 ENDPOINT_URL = os.environ.get("EXODUS_AWS_ENDPOINT_URL") or None
 
+INDEX_FILENAME = "__exodus_autoindex__"
```

Comment:
Elsewhere you have the name as ".__exodus_autoindex". Should it be the same here?

Reply:
forgot to remove this, line - ty